<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RF Packet Interface</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QPushButton" name="recanPortsButton">
     <property name="text">
      <string>Rescan Ports</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="portCombo">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="connectButton">
     <property name="text">
      <string>Connect</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4" rowspan="2">
    <widget class="QGroupBox" name="txPacketGroupBox">
     <property name="title">
      <string>Transmit Packet</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="addressLabel">
        <property name="text">
         <string>Address:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="addressEdit">
        <property name="inputMask">
         <string>HHHH</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="dataLabel">
        <property name="text">
         <string>Data:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QTextEdit" name="dataEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="transmitButton">
        <property name="text">
         <string>Transmit</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QGroupBox" name="dataEncodingBox">
        <property name="title">
         <string>Data Encoding</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QRadioButton" name="asciiRadioButton">
           <property name="text">
            <string>ASCII</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="hexRadioButton">
           <property name="text">
            <string>Hex</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
